src/DesignKit.tsx

```diff
@@ -2632,106 +2634,291 @@ const TooltipsSection: React.FC = () => {
+          </Column>
+        </Row>
+      </AntDCard>
+      <AntDCard title="Nesting">
         <p>
-          With center-aligned <code>{'<Column>'}</code>s:
+          <code>{'<Column>'}</code>s and <code>{'<Row>'}</code>s can nest arbitrarily
         </p>
-        <Columns>
-          <Column align="center">
-            <Button>Content</Button>
-          </Column>
-          <Column align="center">
-            <Button>Content</Button>
+        <Row>
+          <Column>
+            <Row>
+              <Column>
+                <Card>Base Row, Column 1, Row 1, Column 1</Card>
+              </Column>
+              <Column>
+                <Card>Base Row, Column 1, Row 1, Column 2</Card>
+              </Column>
+            </Row>
           </Column>
-          <Column align="center">
-            <Button>Content</Button>
+          <Column>
+            <Row>
+              <Column>
+                <Card>Base Row, Column 2, Row 1, Column 1</Card>
+              </Column>
+              <Column>
+                <Card>Base Row, Column 2, Row 1, Column 2</Card>
+              </Column>
+            </Row>
+            <Row>
+              <Column>
+                <Card>Base Row, Column 2, Row 2, Column 1</Card>
+              </Column>
+              <Column>
+                <Card>Base Row, Column 2, Row 2, Column 2</Card>
+              </Column>
+            </Row>
           </Column>
-        </Columns>
+        </Row>
+      </AntDCard>
+    </ComponentSection>
+  );
+};
+
+const ColumnSection: React.FC = () => {
+  return (
+    <ComponentSection id="Column" title="Column">
+      <AntDCard>
         <p>
-          With right-aligned <code>{'<Column>'}</code>s:
+          A <code>{'<Column>'}</code> wraps child components to be displayed in a vertical column.
+          <br />
+          Also see <a href={`#${ComponentTitles.Row}`}>Row</a>.
+        </p>
+      </AntDCard>
+      <AntDCard title="Usage">
+        <p>
+          The content within a <code>{'<Column>'}</code> can be aligned according to an{' '}
+          <code>{'align'}</code> value.
+        </p>
+        <strong>Row with 3 left-aligned (default) columns:</strong>
+        <Row>
+          <Column align="left">Left-aligned content</Column>
+          <Column align="left">Left-aligned content</Column>
+          <Column align="left">Left-aligned content</Column>
+        </Row>
+        <strong>Row with 3 center-aligned columns:</strong>
+        <Row>
+          <Column align="center">Center-aligned content</Column>
+          <Column align="center">Center-aligned content</Column>
+          <Column align="center">Center-aligned content</Column>
+        </Row>
+        <strong>Row with 3 right-aligned columns:</strong>
+        <Row>
+          <Column align="right">Right-aligned content</Column>
+          <Column align="right">Right-aligned content</Column>
+          <Column align="right">Right-aligned content</Column>
+        </Row>
```

Comment:
nit: it would be easier to tell what's going on if the columns contained block elements with a border